File: PassedAndFailed.java
import java.util.Scanner;
public class PassedAndFailed {
public static void main(String[]args){

   Scanner input = new Scanner(System.in);

	int noOfStudentsPassed = 0;
	int noOfStudentsFailed = 0;
	
	for (int count = 1; count <= 15; count++){

	System.out.print("Enter fifteen scores:");
	int score = input.nextInt();

	
	if (score >= 45){
	noOfStudentsPassed++;
	  }
	
	else{
	noOfStudentsFailed++;
	}

	  }

	System.out.println("Passed is " + noOfStudentsPassed);
	System.out.println("Failed is " + noOfStudentsFailed);

   }

}

File: python/passedandfailed.py
no_of_students_passed = 0
no_of_students_failed = 0
	
for score in range (1, 15):
	score = int(input("Enter fifteen scores:"))
	if score >= 45:
		no_of_students_passed = no_of_students_passed + 1

	else:
		no_of_students_failed += 1
	
print('Passed is , ',no_of_students_passed)
print('Failed is , ',no_of_students_failed)

